src/Radicle/Internal/Storage.hs

```diff
@@ -34,7 +35,7 @@ data StorageBackend m = StorageBackend
 
 -- | Send a list of expressions to a chain identified by the first
 -- argument.
-type StorageSend m = Text -> Seq Value -> m (Either Text ())
+type StorageSend m = Text -> Seq Value -> m (Either ServantError ())
```

Comment:
With this change the `Storage` module becomes dependent on Servant, but the module should be generic. To solve this the error formattaing needs to happen in `HttpStorage`.

Reply:
@geigerzaehler done.